src/tram/tram/management/commands/otxdata.py

```diff
@@ -0,0 +1,103 @@
+import json
```

Comment:
I would prefer that this be an extension/modification of the `loaddata` command vs. being a new command.

Perhaps:
* `manage.py loaddata` - Loads all training data
* `manage.py loaddata <label>` loads specific training data (bootstrap, otx, etc)